src/main.zig

```diff
@@ -116,6 +127,31 @@ export fn occ_do(data: [*]const u8, length: usize, outBuffer: [*]u8, outBufferSi
     return returnLength;
```

Comment:
Where is this variable defined?

Reply:
this part did not change and is defined and sed in line 124, 125